modules/options-for-notebook-server-environments.adoc

```diff
@@ -314,15 +389,15 @@ a| * Python 3.9
 
 ifdef::upstream[]
 | RStudio Server
-| 2023.2 (Recommended)
+| 2023.2  (Recommended)
 a| * Python 3.9
 * R 4.3
 endif::[]
 
 ifndef::upstream[]
 ifdef::cloud-service[]
 | RStudio Server (Technology preview)
-| 2023.2 (Recommended)
+| 2023.2
```

Comment:
hm, I'm thinking why we remove `Recommended` for the cloud-service and we keep Recommended for the self-managed above, or do I understand it wrong?

Reply:
No, this should still be (Recommended). Will wait to see if we want to include RStudio in this table and correct if we keep it.